packages/contracts/hooks/ens.tsx

```diff
@@ -1,43 +0,0 @@
-import { useMemo } from 'react';
-import type { Address } from 'viem';
-import { normalize } from 'viem/ens';
-import { useConfig, useEnsAvatar, useEnsName } from 'wagmi';
-import { ethereum } from '../chains';
-
-type UseEnsType = {
-  /** The ENS name of the wallet. */
-  ensName?: string | null;
-  /** The ENS avatar of the wallet. */
-  ensAvatar?: string | null;
-};
-
-export function useEns({ address, enabled }: { address?: Address; enabled: boolean }): UseEnsType {
-  const wagmiConfig = useConfig();
-
-  const ensConfig = useMemo(() => {
-    return {
-      config: {
-        ...wagmiConfig,
-        chains: [ethereum] as const,
-      },
-      chainId: ethereum.id,
-      universalResolverAddress: ethereum.contracts.ensUniversalResolver.address,
-    };
-  }, [wagmiConfig]);
-
-  const { data: ensName } = useEnsName({
-    query: { enabled },
-    address,
-    ...ensConfig,
-  });
-
-  const normalizedEnsName = useMemo(() => (ensName ? normalize(ensName) : undefined), [ensName]);
-
-  const { data: ensAvatar } = useEnsAvatar({
```

Comment:
No more ENS? Or are we getting it for free from another package?

Reply:
getting it from [@walletsheet/core](https://github.com/web3sheet/web3sheet/blob/main/packages/core/hooks/useEns.tsx) via `useWallet`